### ISSUE TITLE
Q: How to add audiobooks?

### ISSUE THREAD

**Issue by maichai** [ejurgensen/forked-daapd#1202]
I have the following folders:
`/music/Music`
`/music/Podcasts`
`/music/Audiobooks`

I tried the following configs (after changing config and before starting the container I removed the db and cache):

Trial 1:
```
	directories = { "/music/Music", "/music/Audiobooks", "/music/Podcasts" }
	podcasts = { "/music/Podcasts" }
	audiobooks = { "/music/Audiobooks" }
```

Trial 2:
```
	directories = { "/music/Music", "/music/Audiobooks", "/music/Podcasts" }
	podcasts = { "/Podcasts" }
	audiobooks = { "/Audiobooks" }
```

Trial 3:
```
	directories = { "/music" }
	podcasts = { "/Podcasts" }
	audiobooks = { "/Audiobooks" }
```

In all trials:
Music, Podcasts and Audiobooks were shown as separate folders in the Files section.
The music files were shown properly.
The podcast files were shown in the podcast section.

And here is the problem: in none of the trials:
The audiobook files were not shown in the podcast section (and also no other section).

I am sure I misunderstand how to use the `podcasts =` and `audiobooks =` statements in the config. The explanation on the config did it not make it clear enough for me.

For  the folder structure described above, how should I set my `podcasts` and `audiobooks` statements in the config?
Is any of the trials correct? Then there must be a bug...

**Comment by ejurgensen** [comment]
I think all three methods should work, but the "standard" one is probably number 3, as [you can see here](https://github.com/ejurgensen/forked-daapd/blob/master/forked-daapd.conf.in#L84). I think the reason you are not seeing them might have to do with browser caching. To check that, try this request:
`http://[your_rpi_address]:3689/api/library/artists?media_kind=audiobook`

Do you see the audiobooks now? If so, maybe a Ctrl + F5 in the web interface will make the audiobook tab appear.

**Comment by maichai** [comment]
Thanks, with that URL I can see them. They are also visible in the Web UI now. 

However, I am not convinced that browser caching was the sole problem here, as the Audiobooks playlist in the Apple Music app also was empty, and this does not go through any browser! 
And - lo and behold - the Audiobooks playlist **still** is empty, although the Web UI shows some audio books. 

How to fix this?

**Comment by ejurgensen** [comment]
Yes, the Apple Music app is a different issue. I don't think there is way for you to fix that, because that means diving into how an audiobook is marked towards the Music app, and whether the Music app honors that mark. It's been a while since I looked at that stuff myself, so I don't remember exactly how it is supposed to work. If you want you are welcome to open another issue with this observation.

**Comment by ejurgensen** [comment]
I just checked iTunes (don't have Apple Music), and there the audiobooks are listed like they should. So it would probably be difficult for me to find out why they aren't listed correctly in Apple Music app.

**Comment by maichai** [comment]
There are two places, where the audiobooks could show up, either in the Music app in the Audiobooks playlist, or in the Books app under Audiobooks. Since I never bought any audiobook from Apple's store I do not know what is the right place.

Maybe some other users can provide more information on this.
